src/risedevtool/src/bin/risedev-docslt.rs

```diff
@@ -0,0 +1,240 @@
+// Copyright 2022 Singularity Data
```

Comment:
I hoped this could be part of risinglightdb/sqllogictest instead of vendoring in our repo 🤣